drivers/wifi/nrf700x/zephyr/src/zephyr_twt.c

```diff
@@ -114,6 +122,8 @@ void wifi_nrf_event_proc_twt_setup_zep(void *vif_ctx,
 
 	twt_params.operation = WIFI_TWT_SETUP;
 	twt_params.flow_id = twt_setup_info->info.twt_flow_id;
+	/* Store the negotiated flow id and pass it to user. */
+	vif_ctx_zep->neg_twt_flow_id = twt_setup_info->info.twt_flow_id;
```

Comment:
Where is this being cleared? This should be cleared in both all/specific teardown.